### DISCUSSION TITLE
[求助]急急急！谁能来告诉我这里怎么出去啊！

### DISCUSSION THREAD

**Discussion by qiuledadada** [share121/inter-knot#1854] [其他]
事情是这样的，有天我在六分街的路上走看见了一个老头不知道为什么我突然间就来到了这里，现在似乎出不去了谁能来救救我

![Image_1723531934558](https://github.com/user-attachments/assets/d74c9cfa-be56-4b13-adf7-70e7174ef14b)

**Comment by qiuledadada** [comment]
有人吗？！来个人救救我啊！

**Reply by Doctor-Bright** [reply]
基金会帮不了你，自求多福吧.
                                  -DR‘Bright
审核：{以编辑}
0级权限，<Safe>开放访问

Secure.Contain.Protect

**Comment by gaojiaobei** [comment]
兄啊，你这被送去别的世界了吧

**Comment by shuo-guo** [comment]
先这样，然后那样，最后在这样。听懂了吧？

**Comment by QINGLU-jiezhu** [comment]
我也不知道这么进来了，我身上还有5级卡，地表还等着我开核，急急急

**Comment by qiuledadada** [comment]
哦天……这可真是不幸的一天，我现在一点食物没有尝试着走了一会儿感觉像在兜圈子一样

**Comment by Yitaruose-BaiXu** [comment]
这个委托我接下了

**Comment by jzx-176** [comment]
发布委托吧，5000万丁尼带你回家

**Comment by zhuyaohui114514** [comment]
这个委托我接了

**Comment by Landseasky** [comment]
这个委托我接了

**Comment by AnbulesPeo** [comment]
这个委托。。。我不敢接啊

**Comment by zzzuser46** [comment]
建议重开

**Comment by bbdqz** [comment]
布什,戈门

**Comment by icenow188** [comment]
你那好像不是空洞啊，萝卜没用，我也不知道咋办兄弟自求多福吧

**Comment by Tartaglia770** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >
吃瓜中

**Comment by scp-perseus** [comment]
这个委托我接下了！

**Comment by Zhangshibo-sudo** [comment]
这个委托我接了！！

**Comment by 1ctrl-cv** [comment]
随便找个路口走出去，或者大喊一声安娜看看能不能把斯克兰顿博士喊醒<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
